feat: new guide for publishing Shiny apps to Posit connect servers (#53)
CCBR/HowTos: docs/shiny/1_Development.qmd

Posit offers a "cheatsheet" for referencing aspects of Shiny development:

[Shiny Cheatsheet PDF](shiny.pdf)

## Example Shiny App

There is an example Shiny app for CCBR members to use for learning and experimenting with an already functional app, or for use as a foundation for developing a new app.

To set up the example Shiny app for personal use, please use the following steps.

#### Create a fork

The first step is to fork the [CCBR shiny-example github repo](https://github.com/CCBR/shiny-example) using the "fork" option from the CCBR shiny-example repo.

![Use the Fork option (red rectangle) from the CCBR shiny-example repo](images/fork_screenshot.png)\

#### Clone the forked repo

After the fork is created, clone the fork locally using the github CLI with the URL available in the code tab of the forked repo.

![*How to find the URL to use for cloning the forked repo (red box)*](images/clone_URL.png)

The github CLI command will be:

``` bash
git clone https://github.com/$gh_username/shiny-example.git
```

> Where \$gh_username will replace with your github username.

Reviewer: Recommend making it more clear that the Reader needs to manually replace the variable with their username

```suggestion
Be sure to replace `$gh_username` with your actual github username.
```
